roles.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "type": "object",
+  "properties": {
+    "roleName": {
```

Comment:
Can we take the `roleName` from the file name?

Reply:
Yes, I guess we can do that. So I will remove this as well.

Reply:
We would have to give the file a name anyway right? Any objections to this? @wilhelm-ohman or @JensESecuritas what are your thoughts?

Reply:
I set a roleName as file name and left this property inside but I can remove property and rely only on file name.

Reply:
relying on file name sounds like a good idea

Reply:
Ok, so let's remove the `roleName` then. 

Do we use `displayName` for anything or are they always translated based on the `roleName`?

Reply:
Can we check with FE if they are using it in UserManagement? I guess that might be the only places it is used. I don't think we ever care about it in the BE since it is only for displaying.

Reply:
Maybe they use display name in FE or do we use it in IDP?

Reply:
It's used in FE but as a backup. First they look for translations of role name based on the key `roles.${roleName}.title` and if it can't be found `displayName `is used and if it's empty `roleName`.

I guess we can remove this `displayName `as we should have translations for that always and as a fallback use `roleName`?
What do you think? @mfrinnstrom